docs/UserGuide.md

```diff
@@ -297,6 +297,19 @@ Example:
 * `untagEvent 1 p/2` untags the 2nd person in the contact list from the 1st event in the event list
 * `untagEvent 2 p/2 4 5` untags the 2nd, 4th, 5th person in the contact list from the 2nd event in the event list
 
+### Create mailing list for an event : `mailEvent`
+
+Format: `mailEvent EVENT_INDEX`
+
+* The `EVENT_INDEX` refers to the index number shown in the displayed event list.
+* The `EVENT_INDEX` must be a positive integer 1, 2, 3, …, and it must be within the range of the event list index.
+* The mailing list is saved as a CSV file `[JAR file location]/data/EVENT_TITLE.csv`. The CSV file has 2 columns:
+`Name` and `Email`, representing the name and email for a person in the event.
+
+Example:
+* `mailEvent 2` creates mailing list as a CSV file, the name of the csv file is the same as the title of 2nd event.
+* `mailEvent 4` creates mailing list as a CSV file, the name of the csv file is the same as the title of 4th event.
```

Comment:
Might be better to help the user understand where the rankings come from! (Also, if you commit this suggestion directly might fail checkstyle due to the max number of characters per line)
```suggestion
* `mailEvent 2` creates mailing list as a CSV file, the name of the csv file is the same as the title of 2nd event in the events list.
* `mailEvent 4` creates mailing list as a CSV file, the name of the csv file is the same as the title of 4th event in the events list.
```